crates/completion/src/completions/attribute.rs

```diff
@@ -21,20 +21,15 @@ pub(crate) fn complete_attribute(acc: &mut Completions, ctx: &CompletionContext)
 
     let attribute = ctx.attribute_under_caret.as_ref()?;
     match (attribute.path(), attribute.token_tree()) {
-        (Some(path), Some(token_tree)) if path.to_string() == "derive" => {
-            complete_derive(acc, ctx, token_tree)
-        }
-        (Some(path), Some(token_tree)) if path.to_string() == "feature" => {
-            complete_lint(acc, ctx, token_tree, FEATURES);
-        }
-        (Some(path), Some(token_tree))
-            if ["allow", "warn", "deny", "forbid"]
-                .iter()
-                .any(|lint_level| lint_level == &path.to_string()) =>
-        {
-            complete_lint(acc, ctx, token_tree.clone(), DEFAULT_LINT_COMPLETIONS);
-            complete_lint(acc, ctx, token_tree, CLIPPY_LINTS);
-        }
+        (Some(path), Some(token_tree)) => match path.to_string().as_str() {
```

Comment:
Nit: the allocation here can be avoided with `path.syntax().text()`, but then match no longer works (or you need to do `s if s == "derive" => ...` etc.).

Again, not sure it's worth it.